webpack.dev.config.js

```diff
@@ -77,17 +77,18 @@ module.exports = {
         test: /\.(sa|sc|c)ss$/,
         use: [
           { loader: 'style-loader' },
-          { loader: MiniCssExtractPlugin.loader },
+          { loader: MiniCssExtractPlugin.loader, options: { esModule: false } },
```

Comment:
Could you please explain why we need this `esModule: false` now?

Reply:
Maybe this quirk is no longer needed when also updating [mini-css-extract-plugin](https://www.npmjs.com/package/mini-css-extract-plugin) to a more recent version? I will give it a try...

Reply:
But I don't know exactly what's its purpose, if issue is still there and you think it's safe to disable it, go ahead.

Reply:
> This quirk is no longer needed when also updating [mini-css-extract-plugin](https://www.npmjs.com/package/mini-css-extract-plugin) to a more recent version.

Implemented by replacing bef815f533 with c726626123. Thanks.